Ginger/Ginger/Activities/UCValueExpression.xaml.cs

```diff
@@ -74,7 +75,7 @@ public void Init(Context context, object obj, string AttrName, bool isVENeeded =
                 LastCol.Width = new GridLength(55);
                 BrowseButton.Visibility = Visibility.Visible;
                 this.fileType = fileType;
-
+                this.specialFolder = specialFolder;
```

Comment:
rename it from specialfolder to Rootfolder if that is what it is used for, also I don't think someone can define it, can you tell me a real life example for usage?

Reply:
e.g In case of java , i want to browse java bin folder, so i can directly give root directory of "C:\Program Files (x86)"